scripts/release.js

```diff
@@ -167,9 +173,16 @@ if (!fs.existsSync('dependency-check-app')){
 process.chdir(path.join(__dirname, '../dependency-check-app'));
 spawnOrFail('npm', ['init -y']);
 spawnOrFail('npm', ['install react react-dom']);
-spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion} styled-components styled-system`]);
+spawnOrFail('npm', [
+  `install amazon-chime-sdk-js@${updatedSdkVersion} styled-components styled-system`,
+]);
 spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
```

Comment:
Just wondering - have we removed all yalc references in our code? Not this one in particular, since it's a separate test, but in general?

Reply:
Nope, I don't think we have removed all yalc references.  There are two more lines contain yalc in this release file

Reply:
I only see two 'yalc' references in this file. Those are fine, we can leave those.